### ISSUE TITLE
[REQ/Off topic] Custom A, B, C shots?

### ISSUE THREAD

**Issue by tommiehansen** [obs1dium/FocusBracket#10]
Would it be possible to do this?
1. Set camera settings to 'A', click button to confirm
2. Set ... to 'B', click to confirm
3. Set 'C', click to confirm
4. Trigger sequence

'Settings' refer to camera settings which basically can be anything the camera is set to (focus point, wb, f-stop etc etc).

I also have added a specific post about this, or the logic behind it, on community.sony.com but that Sony would include such things in FW updates or even as an app is far fetched.

Detailed post on community.sony.com:
http://community.sony.com/t5/Join-the-Inspiration/Advanced-Bracketing/idi-p/542423

It would seem that your Focus Bracket app handles and does similar things to what i've wanted for quite some time. The big difference is that by doing it "my" way one basically just set settings between shot A, B and C which basically mean that one could bracket _anything_, not just focus or exposure. Since the idea is that the app itself don't care what one brackets one could even combine focus, exposure, wb, shutter speeds or anything that is a setting and the app system got access to.

**Comment by obs1dium** [comment]
That's certainly possible, but also a huge amount of work....

**Comment by tommiehansen** [comment]
Ok, i thought you sort of had it down since you save A and B (focus distance) in the current app. The way i proposed the other things is at its core to be able to save more things and have more A's and B's (a,b,c,d,e,f,g,h,i.. and so on). Making it something fully fledged is ofc another thing.

**Comment by obs1dium** [comment]
The main problem is that you don't have access to the native menus and button behaviors and would have to re-implement all of that.

**Comment by tommiehansen** [comment]
Yeah, i've understood that's the case by checking all the classes and seeing that every app have it's own classes for doing anything. Quite limiting and time consuming unfortunately like you say..

I would just create a fork and do it myself but i really don't understand JAVA at all and find it a bit alien even though i'm just to OOP via my occupation as a webdesigner and art director (js, php etc).

Btw -- i've tested it with A7r II + FE 16-35 and can confirm that it's working. It can "lock up" sometimes for some reason i don't know (it could have just been that i had the wrong settings or something). But else it works fine and as one would expect. Due to the fast loading times of the app it isn't cumbersome to quickly exist and enter.
